.github/workflows/olm_tests.yaml

```diff
@@ -0,0 +1,125 @@
+name: OLM-Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+env:
+  GO_VERSION: '1.18'
```

Comment:
This should be part of dependencies.yaml:

```yaml
dependencies:
  - name: go
    version: 1.18
    refPaths:
```

Reply:
thanks, fixed.